src/AutoMapper/MapperConfiguration.cs

```diff
@@ -457,14 +465,21 @@ private TypeMap FindClosedGenericTypeMapFor(TypePair initialTypes, TypePair type
 
         internal struct MapperFuncs
         {
-            public Delegate Typed { get; }
+            public Delegate Typed { get; }
+
+            public LambdaExpression TypedExpression { get; }
```

Comment:
Remove this? BuildExecutionPlan would achieve almost the same thing.

Reply:
These two were for debugging purposes. I can't see the original expression for `Mapper.Map`, only type maps. This lets me get at the expression used for cached map plans.

Reply:
I'm not sure what you mean. BuildExecutionPlan(typeof(int), typeof(int)) shows me the plan for that mapper. Without affecting things at runtime.

Reply:
Derp. Forgot about that.